src/Wpf/Prism.Wpf/PrismBootstrapperBase.cs

```diff
@@ -0,0 +1,216 @@
+﻿using Prism.Events;
+using Prism.Ioc;
+using Prism.Logging;
+using Prism.Modularity;
+using Prism.Mvvm;
+using Prism.Regions;
+using Prism.Regions.Behaviors;
+using Prism.Services.Dialogs;
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+
+namespace Prism
+{
+    public abstract class PrismBootstrapperBase
+    {
+        IContainerExtension _containerExtension;
+        IModuleCatalog _moduleCatalog;
+
+        /// <summary>
+        /// The dependency injection container used to resolve objects
+        /// </summary>
+        public IContainerProvider Container => _containerExtension;
+
+        /// <summary>
+        /// Gets the shell user interface
+        /// </summary>
+        /// <value>The shell user interface.</value>
+        protected Window Shell { get; set; }
+
+
+        public void Run()
+        {
+            InitializeInternal();
+        }
+
+        /// <summary>
+        /// Run the initialization process.
+        /// </summary>
+        void InitializeInternal()
+        {
+            ConfigureViewModelLocator();
+            Initialize();
+            OnInitialized();
+        }
```

Comment:
this seems silly to me... really InitializeInternal is the Run... don't really see a point to having a public Run that is only calling an InitializeInternal

Reply:
Really just for consistency between the two approaches.